### PR TITLE
Fix for IE

### PR DESCRIPTION
**Pull request by acierto**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-    "build": "babel src --out-dir lib --presets=es2015,stage-2"
+    "build": "babel src --out-dir lib"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/stefalda/react-localization.git"
   },
   "keywords": [
     "react-component",
+    "react-component",
```
Comment (henrikra) [inline package.json]: Why this is second time here?

File: lib/utils.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ function getInterfaceLanguage() {
 function validateTranslationKeys(translationKeys) {
   var reservedNames = ['_interfaceLanguage', '_language', '_defaultLanguage', '_defaultLanguageFirstLevelKeys', '_props'];
   translationKeys.forEach(function (key) {
-    if (reservedNames.includes(key)) {
```
Comment (henrikra) [inline lib/utils.js]: Why this is removed if you are using `array-includes` plugin?

Comment (acierto) [inline lib/utils.js]: Because this is not sources, but transpiled code.
This pitfall doesn't affect this code, what you can actually see from this file.

**Comment by stefalda** [comment]
Hi, I've decided to simply remove the array.includes and opt for indexOf... it wasn't necessary.
Anyway I've updated the package.json file upgrading version numbers.
Thank you

File: lib/utils.js (hunk 2 above)
Comment (henrikra) [reply lib/utils.js]: Oh I did not notice that this was on `lib` folder xD

**Pull request closed without merging**